Library/Formula/mono.rb

```diff
@@ -30,7 +34,13 @@ def install
       --prefix=#{prefix}
       --enable-nls=no
     ]
-    args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
+
+    if build.with? "64bit"
+      args << "--build=x86_64-apple-darwin"
+    else
+      # no option specified
+      args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
```

Comment:
Actually, I'm on a platform that defaults to 64 bits, but I didn't know that until I read the code. With an option, it's obvious. Perhaps I should create a 32-bit option as well to make it obvious?